bookwyrm/templates/user/layout.html

```diff
@@ -28,7 +28,7 @@ <h1 class="title">
         </div>
 
         {% if user.summary %}
-        <div class="column box has-background-white-bis content">
+        <div class="column box has-background-white-bis content preserve-whitespace">
             {{ user.summary|to_markdown|safe }}
```

Comment:
The only tweak this needs is the `spaceless` django tag - that will remove the system-generated whitespace between html tags, which would artificially pad out the expected whitespace in the user bio. You'll just need to add `{% spaceless %}` below line 30, and `{% endspaceless %}` below line 33.

As is:
<img width="610" alt="Screen Shot 2021-07-31 at 10 53 13 AM" src="https://user-images.githubusercontent.com/1807695/127748510-b1f1913b-8304-4a2d-9023-0cf1cb023cba.png">

With `spaceless`:
<img width="609" alt="Screen Shot 2021-07-31 at 10 53 45 AM" src="https://user-images.githubusercontent.com/1807695/127748514-7f15243f-8937-4d2e-b18a-ed3933af925b.png">

Reply:
Done! :smiley: